packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx

```diff
@@ -53,8 +55,13 @@ const CoinmarketPaymentType = ({ method }: Props) => (
                         />
                     </Bg>
                 </IconWrapper>
-                {/* temporary solution - payment mehtod name will be returned by API server to be independent on translations */}
-                <Translation id={`TR_PAYMENT_METHOD_${method.toUpperCase()}` as any} />
+                <div>
```

Comment:
```suggestion
                <>
```
`div` maybe replaced by react fragment

Reply:
Not done, the `div` has visual function